workspace_host/interface.js

```diff
@@ -967,10 +968,22 @@ function _createContainer(workspace, callback) {
     } else {
         args = args.split(' ');
     }
+
+    let networkMode = 'bridge';
+    if (!workspace.settings.workspace_enable_networking) {
+        if (config.workspaceSupportNoInternet) {
+            networkMode = 'no-internet';
+        } else {
+            logger.verbose('Workspace requested unsupported feature enableNetworking:false');
```

Comment:
Thinking out loud - should we log a warning at sync time if someone tries to sync a no-network workspace to a PL instance that doesn't support it? We can suppress it in dev mode to avoid giving false warnings.

Reply:
Unfortunately it's a bit tricky because it's the workspace hosts that do the enforcing, but the PL webservers that do the syncing. Let's discuss this in person to brainstorm ideas.

Reply:
Ahh - as in, the config option applies to workspace hosts, not the PL server?

Reply:
Exactly